src/hooks/subscription-hooks.js

```diff
@@ -76,6 +77,33 @@ export function useJurorBalancesSubscription(jurorId) {
       movements = [],
     } = jurorData.juror || {}
 
+    const movementsFiltered = movements => {
+      // If the juror is activating from the wallent we need to just show an incoming movement in the Active balance
+      return movements.reduce((previousList, movement, index, originalList) => {
+        const convertedMovement = {
+          ...movement,
+          effectiveTermId: parseInt(movement.effectiveTermId, 10),
+          amount: bigNum(movement.amount),
+        }
+        if (index === 0) {
+          previousList[0] = convertedMovement
+        } else if (
+          ANJMovement[originalList[index - 1].type] ===
+            ANJMovement.Activation &&
+          ANJMovement[movement.type] === ANJMovement.Stake &&
+          originalList[index - 1].createdAt === movement.createdAt
```

Comment:
What do you think of storing a variable for `originalList[index - 1]` like `previousMovement` and use it here so it's more clear?